cypress/integration/calculator-homework-Indre-Kelly/calculator.spec.js

```diff
@@ -0,0 +1,66 @@
+import Calculator from '../../page_objects/calculator_main/Calculator.js';
+
+var calculator = new Calculator;
```

Comment:
Works this way, but still suggesting to add () at the end of line i.e. **var calculator = new Calculator();** (using other technologies such syntax might be a must).